setup.py

```diff
@@ -1,14 +1,30 @@
 from setuptools import setup, find_namespace_packages
 from glob import glob
 
-from villas.controller import __version__ as version
+import os
+import re
+
+
+def get_version():
```

Comment:
I did not fully grasp why this is required. The import of the version variable seems a more elegant solution to me, and is also a common approach I've seen in other projects.

I will approve this PR, because we should migrate away from setup.py anyway, and adopt `pyproject.toml` instead.